molgenis-data-import/src/main/java/org/molgenis/data/export/exception/InvalidEntityIdentifierException.java

```diff
@@ -0,0 +1,26 @@
+package org.molgenis.data.export.exception;
+
+import static java.lang.String.format;
+import static java.util.Objects.requireNonNull;
+
+import org.molgenis.i18n.CodedRuntimeException;
+
+public class InvalidEntityIdentifierException extends CodedRuntimeException {
```

Comment:
Invalid entity identifier or entity type identifier?

Reply:
come to think of it... neither -> EmxIdentifier